inbm-lib/inbm_common_lib/request_message_constants.py

```diff
@@ -25,6 +25,8 @@
 INSTALL_CHECK_FAILURE = "Install check failed"
 ERROR_DURING_INSTALL = "Error during install"
 SOTA_OVERALL_FAILURE = "Overall SOTA status: Failure"
+FOTA_NO_NODES_FAILURE = "ERROR! No eligible nodes found"
```

Comment:
FOTA_NO_NODES_FAILURE, this error message also exists in inbm_vision_lib/request_messaage_constants.py as CONFIGURATION_NO_NODES_FAILURE.  It makes more sense to have it in this file.  Could you remove it from the vision_lib and move the location using that one to use this constant instead.